docs/adr/0013-react-toolchain-for-settings-redesign.md

```diff
@@ -0,0 +1,118 @@
+# React Toolchain for Settings Redesign
+
+- Status: proposed
+- Deciders: Lauren Zugai, Jody Heavener
```

Comment:
I'd say put your name first since you wrote this guy, and probably add @lmorchard since I originally was like meh maybe we don't need CRA! And Les brought me back to like actually we do want it for maintainability and such.